tests/README-tests.md

```diff
@@ -0,0 +1,75 @@
+## CytoSnake Testing General Documentation
+
+<p align="center">
+  <img src="../docs/images/testing_diagram.svg", alt="image showing CytoSnake's Testing Suite">
+</p>
+
+>This diagram illustrates the testing framework of `CytoSnake`, showcasing the main objectives of each core testing module.
+
+Welcome to the testing documentation for `CytosSnake`, where you'll learn the essential principles of crafting effective and insightful tests.
```

Comment:
```suggestion
Welcome to the testing documentation for `CytoSnake`, where you'll learn the essential principles of crafting effective and insightful tests.
```

Also, this sentence implies that the audience for this documentation is developers. Is this true?

If so, it might be worth adding a sentence or two specifying who should read this document and why.